containers/mit-scheme/.devcontainer/library-scripts/mit-scheme-debian.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
```

Comment:
We recommend moving this file up one level into `containers/mit-scheme/.devcontainer/`.

Reply:
Done.